packages/bootstrap/scss/dock-manager/_variables.scss

```diff
@@ -0,0 +1,113 @@
+///  The width of the border around the DockManager component.
+/// @group dock-manager
+$kendo-dock-manager-border-width: 1px !default;
+/// The style of the border around the DockManager component.
+/// @group dock-manager
+$kendo-dock-manager-border-style: solid !default;
```

Comment:
In other components we don't add a variable for border-style - instead it is directly set to the component. Example: 
https://github.com/telerik/kendo-themes/blob/8b90f3aef2d8742d9fea94474dabb38bd7553c1d/packages/default/scss/window/_layout.scss#L7

Reply:
The variable just exposes the possibility of customization. I agree though that consistency is nice, however, in this particular case it is not something that should be removed.